examples/ethereum_implementation.md

```diff
@@ -0,0 +1,163 @@
+# Example Implementation in Ethereum
```

Comment:
# Ethereum Implementation 
A good place for it to live would be docs/ethereum

Reply:
Actually I think `internal/pkg/chains/ethereum/readme.MD` is a better place

Reply:
Could be- thanks for clarifying
Moved to this place.